### PR TITLE
Use vaSyncBuffer in DeviceVAAPI::QueryStatus

### PR DESCRIPTION
**Pull request by aobolensk**

### CODE REVIEW

**Comment by dmitryermilov** [comment]
@aobolensk , please add a description to commit message what intention of this commit is

File: _studio/mfx_lib/encode_hw/shared/ehw_device_vaapi.cpp [hunk 1]
```diff
@@ -339,16 +339,21 @@ mfxStatus DeviceVAAPI::Init(
 mfxStatus DeviceVAAPI::QueryStatus(DDIFeedback& ddiFB, mfxU32 id)
 {
     MFX_CHECK(!ddiFB.Get(id), MFX_ERR_NONE);
+    mfxStatus sts;
```
Comment (mgonchar) [inline _studio/mfx_lib/encode_hw/shared/ehw_device_vaapi.cpp:342]: nit: I'd rather left declaration on both lines: 348 and 355

Comment (aobolensk) [reply _studio/mfx_lib/encode_hw/shared/ehw_device_vaapi.cpp]: Done

**Review by dmitryermilov — APPROVED** [review]

File: _studio/mfx_lib/encode_hw/shared/ehw_device_vaapi.cpp [hunk 2]
```diff
@@ -339,16 +339,20 @@ mfxStatus DeviceVAAPI::Init(
 mfxStatus DeviceVAAPI::QueryStatus(DDIFeedback& ddiFB, mfxU32 id)
 {
     MFX_CHECK(!ddiFB.Get(id), MFX_ERR_NONE);
+    auto itMB = std::find_if(ddiFB.ExecParam.begin(), ddiFB.ExecParam.end(), DDIExecParam::IsFunction<VAFID_MapBuffer>);
```
Comment (dvrogozh) [inline _studio/mfx_lib/encode_hw/shared/ehw_device_vaapi.cpp:342]: which encoders are affected by this common code? is vp9 touched?

Comment (aobolensk) [reply _studio/mfx_lib/encode_hw/shared/ehw_device_vaapi.cpp:342]: HEVC is affected here

Comment (dvrogozh) [reply _studio/mfx_lib/encode_hw/shared/ehw_device_vaapi.cpp:342]: ok, thx.

**Review by dvrogozh — APPROVED** [review]

**Review by dvrogozh — APPROVED** [review]

**Pull request merged**